lib/sky-pages-bootstrap-generator.js

```diff
@@ -4,16 +4,11 @@
 function getBootstrap(skyPagesConfig) {
   let bootstrap = '';
 
-  if (skyPagesConfig.auth || skyPagesConfig.omnibar) {
-    bootstrap =
-`import { SkyAppBootstrapper } from '${skyPagesConfig.runtimeAlias}';
-
-SkyAppBootstrapper.bootstrapConfig = {
-  omnibar: ${JSON.stringify(skyPagesConfig.omnibar)},
-  auth: ${JSON.stringify(skyPagesConfig.auth)}
-};
-
-`;
+  if (skyPagesConfig.packages) {
+    bootstrap = `
+      import { SkyAppBootstrapper } from '${skyPagesConfig.runtimeAlias}';
+      SkyAppBootstrapper.bootstrapConfig = ${JSON.stringify(skyPagesConfig.packages)};
```

Comment:
Fair enough. I'll revert to a more deliberate approach (your tests also would have to be rewritten to accommodate, so probably not worth the effort).